src/NuGet.Core/NuGet.Build.Tasks/Properties/AssemblyInfo.cs

```diff
@@ -3,4 +3,6 @@
 
 using System.Runtime.CompilerServices;
 
+[assembly: InternalsVisibleTo("NuGet.Build.Tasks.Console, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
```

Comment:
You probably don't need this. 
We can take direct dependencies on NuGet.Build.Tasks as it's not something we track the public APIs for.

Reply:
This is confusing because `NuGet.Build.Tasks.Test` is listed below with `InternalsVisibleTo`.  Should I make the `StaticGraphRestoreArgument` public?

Reply:
It's ok to internalsvisible to a test, but this is a production assembly. 
We decided not to monitor the APIs for NuGet.Build.Tasks because it's an overkill anyways.

Reply:
Okay I've made `StaticGraphRestoreArguments` public and removed this